docs/components/Chips/Chips.stories.mdx

```diff
@@ -144,3 +144,34 @@ colors for parity with Jobber's mobile platform, but we will not be implementing
 this is for the web components as of this first iteration of our web Chip. The
 mobile use-case is very specific to a filtering use case that we as of now do
 not anticipate will be required for the web.
```

Comment:
L142-146 are now out of date and should either be removed, or reflect the new functionality being added in this PR

Reply:
right. yeah I can delete these.

the notes about the CSS overrides wouldn't really belong here. they are being applied to standalone Chip, so I'll make note of them over in the stories for that component.

though that is somewhat awkward, since we know the case where someone wants to override this **is** a `Chips`. looking... maybe I'll write them there and then link to them from `Chips`

with that in mind, I feel like I should remove the `active` word from the variable. that's only applicable in the `Chips` context, but it belongs to `Chip` generically.